src/main/java/moriyashiine/aylyth/common/world/ModDamageSources.java

```diff
@@ -32,6 +33,10 @@ public DamageSource soulRip(PlayerEntity player) {
 		return new DamageSource(damageTypeRegistry.entryOf(ModDamageTypeKeys.SOUL_RIP), player);
 	}
 
+	public DamageSource blight() {
```

Comment:
Would you mind rewriting this with a `@Nullable PlayerEntity` parameter? This way we can attribute the damage back to a given player if possible.

Reply:
pushing it in a future commit